README.md

```diff
@@ -1,11 +1,12 @@
-# CakePHP Framework
-
+<p align="center"><a href="https://cakephp.org" target="_blank"><img src="https://cakephp.org/img/trademarks/logo-1.jpg"></a></p>
```

Comment:
This will work, centering is not required:
```
![CakePHP Logo](https://cakephp.org/img/trademarks/logo-1.jpg)
```

This would fix a lot more of ugly:
```
![CakePHP Logo](https://cakephp.org/img/cake-logo.svg)
```
However for me embedding svg in markdown on github-comments or MacDown.app did not work. Does it for you?